management/src/main/java/org/ehcache/management/ManagementRegistryServiceConfiguration.java

```diff
@@ -42,4 +42,8 @@
    */
   Collection<String> getTags();
 
+  /**
+   * @return an identifier used to identify this running instance. It will be the same even if a clustered Ehcache client reconnects (and clientId changes).
+   */
+  String getProductInstanceId();
```

Comment:
Should the term here be this 'abstract'? At this point the 'product' is a known thing not an abstract concept.

Reply:
what do you suggest ?

Reply:
getter renamed ;-)

Reply:
Well at a minimum would 'EhcacheInstanceId' be better - just substituting in the concrete product name? Or maybe EhcacheClientIdentifier?

Reply:
I have pushed: `getCacheManagerInstanceId()`